src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayViewController.swift

```diff
@@ -34,6 +34,15 @@ class DiaryDayViewController: UIViewController, UITableViewDataSource, UITableVi
 
 		view.backgroundColor = .enaColor(for: .darkBackground)
 
+		// a fix for transparent tab bar when there is content on the back
```

Comment:
Would be nice to have more context here. A detailed description and maybe a link to more context.